### DISCUSSION TITLE
map/surf_tequila

### DISCUSSION THREAD

**Discussion by giscus** [Kalekki/SurfHeaven_Extended#4] [Announcements]
# map/surf_tequila

Click the download icon to download surf_tequila. You can also see the top 100 records for the map as well as various other information.

https://surfheaven.eu/map/surf_tequila

**Comment by Kalekki** [comment]
![burpoSpin](https://cdn.7tv.app/emote/60845a505e01df61570a6f1d/4x.webp)
